core/domain/event_services.py

```diff
@@ -287,12 +287,6 @@ def _refresh_registry(cls):
             if inspect.isclass(obj) and issubclass(obj, BaseEventHandler):
                 if obj_name == 'BaseEventHandler':
                     continue
-                if not obj.EVENT_TYPE:
```

Comment:
These tests are shifted to its test file. The tests are called now EventHandlerNameTests

Reply:
Why did you shift them? Seems to me like they fit better here, we should just test that invalid classes aren't accepted?

Reply:
No its a test that will ensure no duplicate event handler exists. This can only be checked in a test file since these lines would become then untestable. Its similar to this PR: https://github.com/oppia/oppia/pull/6710/files#diff-b19129c58e90eccf3f2592d2abd205df

Reply:
Gotcha, makes sense. SGTM.